crates/oxc_linter/src/snapshots/anchor_has_content.snap

```diff
@@ -0,0 +1,48 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 119
+expression: anchor_has_content
+---
+  ⚠ eslint-plugin-jsx-a11y(anchor-has-content): Missing accessible content when using `a` elements.
+   ╭─[anchor_has_content.tsx:1:1]
+ 1 │ <a />
+   ·  ─
+   ╰────
+  help: Provide screen reader accessible content when using `a` elements.
+
+  ⚠ eslint-plugin-jsx-a11y(anchor-has-content): Missing accessible content when using `a` elements.
+   ╭─[anchor_has_content.tsx:1:1]
+ 1 │ <a><Bar aria-hidden /></a>
+   · ──────────────────────────
```

Comment:
This is still pointing to the whole element, not `a` 🤔

Reply:
Here I consider that the code it modifiers is to remove `aria-hidden`, notice the help hint, whether it would be better to annotate the whole `a` element, or just annotate the `Bar `element🤔

Reply:
Ohh, I didn't notice the subtle difference. Thank you for the explanation!